platformcomponents/desktop/toggle-button-overlay.json

```diff
@@ -0,0 +1,50 @@
+{
+  "toggleButtonOverlay": {
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=44859%3A103949&t=p8zl3QPbotGIIFT3-1",
+    "comment": "Applied to circle and pill toggle buttons",
+    "inactive": {
+      "#normal": {
+        "background": "@theme-button-inverted-normal",
+        "border": "@theme-outline-button-normal",
+        "text": "@theme-text-secondary-normal"
+      },
+      "#hovered": {
+        "background": "@theme-button-inverted-hover",
+        "border": "@theme-outline-button-normal",
+        "text": "@theme-text-primary-normal"
```

Comment:
also for the hover and pressed state; it should be @theme-text-secondary-normal

Reply:
The Figma says primary though. Should I continue to change it to secondary?

Reply:
true! sorry, my bad